android-sdk/src/main/java/co/elastic/apm/android/sdk/ElasticApmConfiguration.java

```diff
@@ -122,6 +134,36 @@ public Builder setSignalConfiguration(SignalConfiguration signalConfiguration) {
             return this;
         }
 
+        /**
```

Comment:
In the end, I figured that, at least in Android, setting the filters with a function named `set*` might be better in order to keep consistency with all the other config options @bryce-b @gregkalapos jsyk